src/common/time/WallClock.h

```diff
@@ -0,0 +1,39 @@
+/* Copyright (c) 2018 vesoft inc. All rights reserved.
+ *
+ * This source code is licensed under Apache 2.0 License,
+ * attached with Common Clause Condition 1.0, found in the LICENSES directory.
+ */
+
+#ifndef COMMON_TIME_WALLCLOCK_H_
+#define COMMON_TIME_WALLCLOCK_H_
+
+#include "base/Base.h"
+#include <time.h>
+
+namespace nebula {
+namespace time {
+
+/**
+ * The class provides several utility methods to allow fast access the time
+ *
+ * The *fast* versions are way faster than the *slow* versions, but not as
+ * precise as the *slow* versions. Choose wisely
+ *
+ */
+class WallClock final {
+public:
+    WallClock() = delete;
+
+    static int64_t slowNowInSec();
```

Comment:
Do we need slowXXX any more?

Why not change TimeUtils Implementation directly?  Why use another interface?

Reply:
The name ```TimeUtils``` is too general. It's better to make names more specific and meaningful. What we need here is to get the wall clock time

```slowXXX()``` is more accurate than ```fastXXX()```, but slower, so we keep them for scenarios that require accurate time

Reply:
> slowXXX() is more accurate than fastXXX(), but slower, so we keep them for scenarios that require accurate time

Sounds reasonable.  

> The name TimeUtils is too general. It's better to make names more specific and meaningful. What we need here is to get the wall clock time

We'd better use another name to avoid copyright infringement

Reply:
WallClock is not copyrighted. It's been used in a lot of places